lib/Makefile

```diff
@@ -148,17 +154,17 @@ install: lib liblz4.pc
 	@$(INSTALL_DATA) liblz4.pc $(DESTDIR)$(PKGCONFIGDIR)/
 	@echo Installing libraries
 ifeq ($(BUILD_STATIC),yes)
-	@$(INSTALL_LIB) liblz4.a $(DESTDIR)$(LIBDIR)/liblz4.a
+	@$(INSTALL_DATA) liblz4.a $(DESTDIR)$(LIBDIR)/liblz4.a
```

Comment:
My understanding is that a dynamic library is considered an "executable",
so it must use `INSTALL_PROGRAM` which kind of `chmod 755` the file,
while a static library is just an archive with `*.o` files into it,
so it's plain data, read at link stage.